pkg/models/repository_rpm_test.go

```diff
@@ -9,9 +9,9 @@ func (s *ModelsSuite) TestRepositoriesRpmsValidations() {
 	const spName = "testrepositoriesrpmsvalidations"
 
 	testRepository := Repository{
-		URL:           "https://my-repository.com",
-		LastReadTime:  nil,
-		LastReadError: nil,
+		URL:                    "https://my-repository.com",
```

Comment:
How about using more [RFC 6761](https://www.rfc-editor.org/rfc/rfc6761) domain names? i.e. "example.com.", "example.net.",  "example.org.", and any names falling within those domains